diff -urpN linux-3.14.1-untouched/arch/x86/syscalls/syscall_32.tbl linux-3.14.1/arch/x86/syscalls/syscall_32.tbl
--- linux-3.14.1-untouched/arch/x86/syscalls/syscall_32.tbl	2014-04-14 15:50:10.000000000 +0200
+++ linux-3.14.1/arch/x86/syscalls/syscall_32.tbl	2016-03-16 22:02:26.528350120 +0100
@@ -359,3 +359,4 @@
 350	i386	finit_module		sys_finit_module
 351	i386	sched_setattr		sys_sched_setattr
 352	i386	sched_getattr		sys_sched_getattr
+353	i386	lin_hello		sys_lin_hello
diff -urpN linux-3.14.1-untouched/arch/x86/syscalls/syscall_64.tbl linux-3.14.1/arch/x86/syscalls/syscall_64.tbl
--- linux-3.14.1-untouched/arch/x86/syscalls/syscall_64.tbl	2014-04-14 15:50:10.000000000 +0200
+++ linux-3.14.1/arch/x86/syscalls/syscall_64.tbl	2016-03-16 22:01:15.948351285 +0100
@@ -322,6 +322,7 @@
 313	common	finit_module		sys_finit_module
 314	common	sched_setattr		sys_sched_setattr
 315	common	sched_getattr		sys_sched_getattr
+316	common	lin_hello		sys_lin_hello
 
 #
 # x32-specific system call numbers start at 512 to avoid cache impact
diff -urpN linux-3.14.1-untouched/kernel/lin_hello.c linux-3.14.1/kernel/lin_hello.c
--- linux-3.14.1-untouched/kernel/lin_hello.c	1970-01-01 01:00:00.000000000 +0100
+++ linux-3.14.1/kernel/lin_hello.c	2016-03-16 22:04:12.804348365 +0100
@@ -0,0 +1,8 @@
+#include <linux/syscalls.h> /* For SYSCALL_DEFINE */
+#include <linux/kernel.h>
+
+SYSCALL_DEFINE0(lin_hello)
+{
+  printk(KERN_DEBUG "Hello World\n");
+  return 0;
+}
diff -urpN linux-3.14.1-untouched/kernel/Makefile linux-3.14.1/kernel/Makefile
--- linux-3.14.1-untouched/kernel/Makefile	2014-04-14 15:50:10.000000000 +0200
+++ linux-3.14.1/kernel/Makefile	2016-03-16 22:05:58.704346617 +0100
@@ -10,7 +10,7 @@ obj-y     = fork.o exec_domain.o panic.o
 	    kthread.o sys_ni.o posix-cpu-timers.o \
 	    hrtimer.o nsproxy.o \
 	    notifier.o ksysfs.o cred.o reboot.o \
-	    async.o range.o groups.o smpboot.o
+	    async.o range.o groups.o smpboot.o lin_hello.o
 
 ifdef CONFIG_FUNCTION_TRACER
 # Do not trace debug files and internal ftrace files
diff -urpN linux-3.14.1-untouched/security/tomoyo/builtin-policy.h linux-3.14.1/security/tomoyo/builtin-policy.h
--- linux-3.14.1-untouched/security/tomoyo/builtin-policy.h	1970-01-01 01:00:00.000000000 +0100
+++ linux-3.14.1/security/tomoyo/builtin-policy.h	2016-03-16 22:11:33.808341083 +0100
@@ -0,0 +1,12 @@
+static char tomoyo_builtin_profile[] __initdata =
+"";
+static char tomoyo_builtin_exception_policy[] __initdata =
+"initialize_domain /sbin/modprobe from any\n"
+"initialize_domain /sbin/hotplug from any\n"
+"";
+static char tomoyo_builtin_domain_policy[] __initdata =
+"";
+static char tomoyo_builtin_manager[] __initdata =
+"";
+static char tomoyo_builtin_stat[] __initdata =
+"";
diff -urpN linux-3.14.1-untouched/security/tomoyo/policy/exception_policy.conf linux-3.14.1/security/tomoyo/policy/exception_policy.conf
--- linux-3.14.1-untouched/security/tomoyo/policy/exception_policy.conf	1970-01-01 01:00:00.000000000 +0100
+++ linux-3.14.1/security/tomoyo/policy/exception_policy.conf	2016-03-16 22:11:22.168341276 +0100
@@ -0,0 +1,2 @@
+initialize_domain /sbin/modprobe from any
+initialize_domain /sbin/hotplug from any
